### PR TITLE
Add RVC scripts

### PR DESCRIPTION
**Pull request by duyen-tran**
python scripts added:
TC_RVCCLEANM_1_2.py
TC_RVCCLEANM_2_1.py
TC_RVCCLEANM_3_2.py
TC_RVCRUNM_1_2.py
TC_RVCRUNM_2_1.py
TC_RVCRUNM_3_2.py
TC_RVCOPSTATE_2_1.py
TC_RVCOPSTATE_2_3.py

yaml scripts added:
-Test_TC_RVCCLEANM_1_1.yaml
-Test_TC_RVCRUNM_1_1.yaml

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_RVCCLEANM_1_1.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023 Project CHIP Authors
+# Copyright (c) 2021 Project CHIP Authors
```
Comment (plauric) [inline src/app/tests/suites/certification/Test_TC_RVCCLEANM_1_1.yaml:1]:
```suggestion
# Copyright (c) 2023 Project CHIP Authors
```

File: src/app/tests/suites/certification/Test_TC_RVCRUNM_1_1.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023 Project CHIP Authors
+# Copyright (c) 2021 Project CHIP Authors
```
Comment (plauric) [inline src/app/tests/suites/certification/Test_TC_RVCRUNM_1_1.yaml:1]:
```suggestion
# Copyright (c) 2023 Project CHIP Authors
```

**Review by plauric — APPROVED** [review]

File: src/python_testing/TC_RVCRUNM_1_2.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+            runTags = [tag.value for tag in Clusters.RvcRunMode.Enums.ModeTag
+                       if tag is not Clusters.RvcRunMode.Enums.ModeTag.kUnknownEnumValue]
+
+            for m in supported_modes:
+                for t in m.modeTags:
+                    is_mfg = (0x8000 <= t.value and t.value <= 0xBFFF)
+                    asserts.assert_true(t.value in commonTags.keys() or t.value in runTags or is_mfg,
+                                        "Found a SupportedModes entry with invalid mode tag value!")
+
```
Comment (cecille) [inline src/python_testing/TC_RVCRUNM_1_2.py:91]: There's something in the test plan about a cleaning mode tag - 
Verify that at least one ModeOptionsStruct entry includes the Cleaning(0x4001) mode tag in the ModeTags field

Verify that none of the ModeOptionsStruct entries include both the Idle(0x4000) mode tag and the Cleaning(0x4001) mode tag in the ModeTags field

I think those checks might be missing.

**Review by cecille — COMMENTED** [review]
If you don't want to fix this pre-TE2, could you please open an issue to address the style comments in the previous PR and also to use the generated enum values?

Also, can you file an issue for implementing the test app in a way that the various settings can be adjusted / used in the CI? would be good to get these running in the CI even with the manual steps. I'm not sure how you do this on a real device, but on the apps, we have fault injection and a fault injection cluster that can be used.

File: src/app/tests/suites/ciTests.json [hunk 4]
```diff
@@ -12,14 +12,24 @@
         "Test_TC_ACL_2_9",
         "Test_TC_ACL_2_10"
     ],
-    "AccessControlEnforcement": ["Test_TC_ACE_1_1", "Test_TC_ACE_1_5"],
-    "BooleanState": ["Test_TC_BOOL_1_1", "Test_TC_BOOL_2_1"],
-    "Binding": ["Test_TC_BIND_1_1"],
+    "AccessControlEnforcement": [
```
Comment (plauric) [inline src/app/tests/suites/ciTests.json:15]: @duyen-tran let's remove the changes from this file for now. Cecille clarified the CI build is caused by something else.

File: src/python_testing/TC_RVCCLEANM_2_1.py [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0000"), "RVCCLEANM.S.A0000 must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0001"), "RVCCLEANM.S.A0001 must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.C00.Rsp"), "RVCCLEANM.S.C00.Rsp must be supported")
+        asserts.assert_true(self.check_pics("RVCCLEANM.S.C01.Tx"), "RVCCLEANM.S.C01.Tx must be supported")
+
+        attributes = Clusters.RvcCleanMode.Attributes
+
+        self.print_step(1, "Commissioning, already done")
+
+        self.print_step(2, "Read SupportedModes attribute")
```
[the remark this answers is not in the document]
Comment (plauric) [reply src/python_testing/TC_RVCCLEANM_2_1.py:67]: @cecille https://github.com/CHIP-Specifications/chip-test-plans/pull/3144

**Review by cecille — APPROVED** [review]

**Pull request merged**